amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,74 @@
+= Amazon SQS - Upgrade the Connector to Version 5.6 - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-upgrade-migrate.adoc
+
+Upgrade Anypoint Connector for Amazon SQS to take advantage of the ability to run an app on a single node or on all nodes of a cluster.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.5.0 |5.6.0
+|5.4.3 |5.6.0
+|===
+
+== Changes in This Release
+
+* Cluster node support was added back to the Receive Messages input source and works the same as it did in version 5.4.3. The input source listens for messages on all cluster nodes by default, but you can change this behavior by using the `Primary node only` field on the `Advanced` sub-tab on the `Receive messages` tab. For more information, see xref:amazon-sqs-connector-config-topics.adoc#node-behavior[Receiving Messages in Cluster Mode].
+
+* Version 5.5.0 was deprecated because cluster mode support was accidentally removed from that version.
```

Comment:
Accidentally? Erroneously?

Reply:
"Accidentally" is wording Gaston used. I think it's better than "Erroneously"  because it shows it was unintentional.